fecgov/fec-pattern-library
# Resolve vulerability in ajv@6.2.12

## Summary of changes

- Addresses #164 
- Pin ajv@6.2.13 to resolve snyk vulnerability


## How to test
- requesting at least one-front end reviewer (@rfultz )
- check out fresh master branch
- snyk test --file=package.json (verify vulnerability exists)
- check out this PR
- remove package-lock.json, node_modules
- npm i
-  snyk test --file=package.json (verify vulnerability no longer exists)
